src/client/hooks/analytics.hook.ts

```diff
@@ -149,3 +149,26 @@ export function trackItemClick(
 ) {
   return trackLink(url, linkTitle, [profileTypeDimension(profileType)]);
 }
+
+export function trackDownload(
+  downloadKind: string,
```

Comment:
Wat is een downloadKind? Je geeft in DocumentList.tsx een document.title voor dit gegeven. Ik zou die gegevens, kijkend naar de naamgeving, niet direct aan elkaar relateren.

Reply:
Hernoemt naar; `downloadDescription` het idee is dat we een beschrijving (kunnen) geven van het soort document (vergunning, jaaropgave, etc.)